pandas/tests/plotting/test_converter.py

```diff
@@ -361,3 +361,13 @@ def test_format_timedelta_ticks(self, x, decimal, format_expected):
         tdc = converter.TimeSeries_TimedeltaFormatter
         result = tdc.format_timedelta_ticks(x, pos=None, n_decimals=decimal)
         assert result == format_expected
+
+    @pytest.mark.parametrize("view_interval", [(1, 2), (2, 1)])
+    def test_call(self, view_interval, monkeypatch):
```

Comment:
can you add the issue number as a comment; also can you give the test a meaningful name

Reply:
yeah, will do.